core/kazoo_apps/src/kapps_util.erl

```diff
@@ -207,12 +207,16 @@ account_depth(Account) ->
 %%--------------------------------------------------------------------
 -spec account_has_descendants(ne_binary()) -> boolean().
 account_has_descendants(Account) ->
-    AccountId = kz_util:format_account_id(Account, 'raw'),
+    View = <<"accounts/listing_by_descendants">>,
+    AccountId = kz_util:format_account_id(Account),
     ViewOptions = [{'startkey', [AccountId]}
                   ,{'endkey', [AccountId, kz_json:new()]}
                   ],
-    {'ok', JObjs} = kz_datamgr:get_results(?KZ_ACCOUNTS_DB, <<"accounts/listing_by_descendants">>, ViewOptions),
-    length([JObj || JObj <- JObjs, kz_account:id(JObj) =/= AccountId]) > 0.
+    {'ok', JObjs} = kz_datamgr:get_results(?KZ_ACCOUNTS_DB, View, ViewOptions),
+    [] =/=
+        [JObj || JObj <- JObjs,
+                 kz_account:id(JObj) =/= AccountId
+        ].
```

Comment:
It's just something I came about when fixing something else. No benchmarking involved.
But yeah I should rewrite this with recursion and return ASAP.